Sources/SwiftSyntaxMacroExpansion/MacroReplacement.swift

```diff
@@ -230,18 +286,22 @@ extension MacroDeclSyntax {
       throw DiagnosticsError(diagnostics: visitor.diagnostics)
     }
 
-    return .expansion(definition, replacements: visitor.replacements)
+    return .expansion(definition, replacements: visitor.replacements, genericReplacements: visitor.genericReplacements)
   }
 }
 
 /// Syntax rewrite that performs macro expansion by textually replacing
 /// uses of macro parameters with their corresponding arguments.
 private final class MacroExpansionRewriter: SyntaxRewriter {
   let parameterReplacements: [DeclReferenceExprSyntax: Int]
+  let genericParameterReplacements: [DeclReferenceExprSyntax: Int]
   let arguments: [ExprSyntax]
 
-  init(parameterReplacements: [DeclReferenceExprSyntax: Int], arguments: [ExprSyntax]) {
+  init(parameterReplacements: [DeclReferenceExprSyntax: Int],
+       genericReplacements: [DeclReferenceExprSyntax: Int],
+       arguments: [ExprSyntax]) {
```

Comment:
Could you format this using swift-format? https://github.com/apple/swift-syntax/blob/main/CONTRIBUTING.md#formatting

CI will reject it otherwise

Reply:
done thanks -- nice to see a swift repo actually use formatting, nice surprise :) 
(We do on all server libs, big fan of forcing it in CI 🙏)

Reply:
Seems to pass, you're right -- I wonder what I messed up back then